app/page.tsx

```diff
@@ -193,31 +236,65 @@ function WalletOverview({ walletInfo }: { walletInfo: WalletInfo }) {
               </Tooltip>
             </TooltipProvider>
           </div>
+          <div>Mainnet ETH</div>
+          <div>
+            <TooltipProvider>
+              <Tooltip>
+                <TooltipTrigger>{getEtherWithPrecison(walletInfo.mainnetEthBalance)}</TooltipTrigger>
+                <TooltipContent>
+                  <p>RAW: {walletInfo.mainnetEthBalance.toString()}</p>
+                </TooltipContent>
+              </Tooltip>
+            </TooltipProvider>
+          </div>
         </div>
         {/* ^ end of balances */}
         <h5 className="text-base font-semibold pt-4">Actions</h5>
         <div className="flex flex-col gap-2">
           <div className="flex flex-wrap items-baseline gap-1.5">
             {/* TODO: sort actions wrt value */}
-            <a>Orbiter.Finance <span className="text-xs text-neutral-400">ZKSync -&gt; Arbitrum</span></a>
-            <Input className='w-24'
-              placeholder={`~ ${getEtherWithPrecison(walletInfo.zkSyncEthBalance)}`}
-              value={actionsState.orbiterZksyncArbitrumValue}
-              onChange={(e: ChangeEvent<HTMLInputElement>) => setActionsState({ ...actionsState, orbiterZksyncArbitrumValue: e.target.value })} />
-            <Button variant="outline" onClick={() => setActionsState({ ...actionsState, orbiterZksyncArbitrumValue: "MAX" })}>Max</Button>
-            <Button disabled={actionsState.orbiterZksyncArbitrumValue === undefined}
-                    onClick={async () => {
-              const txHash = await sendOrbiterTxZksyncToArbitrum(walletInfo.wallet, actionsState.orbiterZksyncArbitrumValue!)
-              setRecentTransactions([
-                {
-                  chain: zkSync,
-                  hash: txHash,
-                  status: "inProgress",
-                  description: `Orbiter: ZKSync -> Arbitrum, ${actionsState.orbiterZksyncArbitrumValue!} ETH`
-                },
-                ...recentTransactions
-              ])
-            }}>Run</Button>
+            <div>
```

Comment:
not sure about this <div>, it can break the flex. Could you please unwrap this from div and wrap BASE action into `<div className="flex flex-wrap items-baseline gap-1.5">`